Binomial-Distribution.ipynb

```diff
@@ -0,0 +1,268 @@
+{
+ "cells": [
```

Comment:
I don't want to change any of the other files, just add the new one.
Unfortunately, you didn't make a branch, so it's hard to fix.